### DISCUSSION TITLE
Migration from v6 to v8 caused memory to blow up

### DISCUSSION THREAD

**Discussion by AnkushSarkar10** [pixijs/pixijs#10592] [v8 General]
I am currently trying to migrate from pixiv6 to v8. I see a huge increase in memory usage. We mostly use textures. Any idea what could be causing this issue?

The memory in dev tools (the js heap) seems fine but the browser (in task manager) is taking up gigs of memory. 

My guess is it might have something to do with these issues
- https://github.com/pixijs/pixijs/issues/10586
- https://github.com/pixijs/pixijs/issues/10578

**Comment by bigtimebuddy** [comment]
@AnkushSarkar10 Can you tell us a little more about your content? What objects are you using or not using?

**Reply by AnkushSarkar10** [reply]
We are using Application, Assets, Spritesheet, Graphics, Container, Text, Renderer, Point, Rectangle

There is no memory spike when loading textures (we load them all at the begining using `await Asset.load(...)` ) or any memory usage decrease when unloading/destroying textures.

The spike in memory seems to occur when graphics/sprites are created and things are drawn.

**Reply by bigtimebuddy** [reply]
Can you isolate if it is just graphics, just sprites or both?

**Reply by AnkushSarkar10** [reply]
both

**Reply by bigtimebuddy** [reply]
Are you calling destroy on graphics/sprite when you no longer need them?

**Reply by AnkushSarkar10** [reply]
yeah we are
